gen/aircraft.py

```diff
@@ -673,6 +686,7 @@ def __init__(
         self.radio_registry = radio_registry
         self.unit_map = unit_map
         self.flights: List[FlightData] = []
+        self.CustomAirSupport = CustomAirSupport()
```

Comment:
And then this becomes:

```
self.air_support = air_support
```

Which is passed to the constructor from `Operation._generate_air_units`.

(`UpperCase` names are types, btw. `snake_case` for members, functions, methods, locals, etc. https://www.python.org/dev/peps/pep-0008/ is roughly what we follow.)